scripts/generate_enhanced_test_data.py

```
@@ -0,0 +1,1147 @@
+
+def create_local_files(target_dir: str = ".", module_set: str = "all"):
+    """
+    Create a local test project containing generated buggy modules for Soulcaster.
+    
+    Creates or replaces the directory at `target_dir`, writes generated module files, a README.md describing included modules, and a requirements.txt. Always writes math_ops.py, string_utils.py, and user_manager.py; if `module_set` is "all" also writes api_client.py, database.py, and cache_manager.py.
+    
+    Parameters:
+        target_dir (str): Filesystem path where the project will be created. If the path exists it will be removed and recreated.
+        module_set (str): Which module set to generate. "basic" creates only the core modules; "all" (default) includes additional advanced modules.
+    """
+def create_local_files(target_dir: str = ".", module_set: str = "all"):
```

Ruff (0.14.8)

Redefinition of unused `create_local_files` from line 796

(F811)